compiler/luci/pass/src/PropagateQParamBackwardPass.cpp

```diff
@@ -15,11 +15,201 @@
  */
 
 #include "luci/Pass/PropagateQParamBackwardPass.h"
+#include "QuantizationUtils.h"
 
 #include <luci/IR/CircleNodes.h>
 #include <luci/IR/CircleNodeVisitor.h>
+#include <luci/Service/Nodes/CircleConst.h>
 #include <luci/Log.h>
 
+#include <cmath>
+
+namespace
+{
+
+void quant_const_values(luci::CircleConst *const_node, float scaling_factor, float zerop,
+                        loco::DataType quant_type)
+{
+  uint32_t size = const_node->size<loco::DataType::FLOAT32>();
+
+  const float scaling_factor_inv = 1.0 / scaling_factor;
+  std::vector<int32_t> quantized_values(size);
+  for (uint32_t i = 0; i < size; ++i)
+  {
+    auto data = static_cast<double>(const_node->at<loco::DataType::FLOAT32>(i));
+    double quantized_float = std::round(data * scaling_factor_inv) + zerop;
+    constexpr auto int_max = static_cast<double>(std::numeric_limits<int32_t>::max());
+    constexpr auto int_min = static_cast<double>(std::numeric_limits<int32_t>::min());
```

Comment:
Not mandatory, but how about applying following change? :D

```suggestion
    constexpr auto int_max_as_double = static_cast<double>(std::numeric_limits<int32_t>::max());
    constexpr auto int_min_as_double = static_cast<double>(std::numeric_limits<int32_t>::min());
```

Type of `int_max` and `int_min` seems `int` but they are actually `double` :)

Reply:
Same with https://github.com/Samsung/ONE/pull/8509#discussion_r813497064. It'll be good to make a PR for renaming.